gpt4all-chat/main.qml

```diff
@@ -14,10 +14,10 @@ import mysettings
 
 Window {
     id: window
-    width: 1280
-    height: 720
-    minimumWidth: 720
-    minimumHeight: 480
+    width: 1920
+    height: 1080
+    minimumWidth: 1280
+    minimumHeight: 720
```

Comment:
Could you explain why we are changing this?

Reply:
The application when it comes up is far smaller by default than other applications of its kind. Further, with the new home page we need more space for the large font and so on.